lib/node_modules/@stdlib/stats/base/dsmeanors/docs/repl.txt

```diff
@@ -33,30 +33,27 @@
     // Standard Usage:
     > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 2.0 ] );
     > {{alias}}( x.length, x, 1 )
-    ~0.3333
+    ~0.333
 
-    // Using `N` and `stride` parameters:
+    // Using `N` and stride parameters:
     > x = new {{alias:@stdlib/array/float32}}( [ -2.0, 1.0, 1.0, -5.0, 2.0, -1.0 ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > var stride = 2;
-    > {{alias}}( N, x, stride )
-    ~0.3333
+    > {{alias}}( 3, x, 2 )
+    ~0.333
 
     // Using view offsets:
     > var x0 = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0 ] );
     > var x1 = new {{alias:@stdlib/array/float32}}( x0.buffer, x0.BYTES_PER_ELEMENT*1 );
-    > N = {{alias:@stdlib/math/base/special/floor}}( x0.length / 2 );
-    > stride = 2;
-    > {{alias}}( N, x1, stride )
-    ~-0.3333
+    > {{alias}}( 3, x1, 2 )
+    ~-0.333
```

Comment:
```suggestion
    ~-0.3333
```

Reply:
Avoid making extraneous changes. Only change what is necessary; otherwise you just add noise to your PR.